test/03-worker.test.js

```diff
@@ -29,3 +30,98 @@ test('worker pipe', async function ({ is, plan, comment, teardown }) {
 
   pipe.write('exit')
 })
+
+test('worker should run directly in a terminal app', async function ({ is, plan, comment, teardown }) {
+  plan(2)
+
+  const helper = new Helper()
+  teardown(() => helper.close(), { order: Infinity })
+  await helper.ready()
+
+  const testId = Math.floor(Math.random() * 100000)
+  comment('Staging harness...')
+  const staging = helper.stage({ channel: `test-${testId}`, name: `test-${testId}`, key: null, dir: harness, cmdArgs: [], dryRun: false, bare: true, ignore: [] })
+  const until = await Helper.pick(staging, [{ tag: 'staging' }, { tag: 'final' }])
+  const { link } = await until.staging
+  await until.final
+
+  const running = await Helper.open(link, { tags: ['exit'] })
+
+  comment('Running worker...')
+  const response = await running.inspector.evaluate(`
```

Comment:
the new convention here is to create a new separate fixtures for each test

the flow is
- Helper.run
- Helper.untilResult
- Helper.untilClose

ref.
https://github.com/holepunchto/pear/blob/main/test/01-smoke.test.js
https://github.com/holepunchto/pear/blob/main/test/04-encrypted.test.js
https://github.com/holepunchto/pear/pull/416/files